src/vfs/public.c

```diff
@@ -124,7 +132,7 @@ void vfs_mkdir_required(const char *path) {
 		log_fatal("%s", vfs_get_error());
 	}
 }
-
+/*
```

Comment:
If you intended to comment out/rewrite this entire function, simply delete the whole thing.